.github/workflows/bump-version-and-publish.yml

```diff
@@ -0,0 +1,54 @@
+# NOTE: taken from https://github.com/MylesBorins/node-osc/blob/main/.github/workflows/bump-version.yml
+
+name: Bump version & Publish
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
You can use `choice` type now :)

See https://github.com/mediamonks/pota/blob/main/.github/workflows/bump-and-publish.yml#L6-L36 for an example

Reply:
Ah will update it! I copied this configuration from the @mediamonks/eslint-config repository :)